### DISCUSSION TITLE
Pinning of all Fossify Apps

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#253] [Ideas]
Hey,

when I'm on the FossifyOrg Start Page on Github, it only shows a few Fossify Apps and it bothers me a bit, that I need to click on all Repositorys to see the all other Fossify Apps.

Could you pin this Apps all too?
Keyboard, Clock, Contacts, Launcher, Draw, Notes, Camera, Flashlight, Voice-Recorder and Calculator. Because it would make it a lot easier for us as Users to directly find the Apps pinned on the Start Page of FossifyOrg Github Site.

**Comment by Aga-C** [comment]
There's a limit, we can only pin 6 repositories.

**Reply by ghost** [reply]
Oh okay, thank you for this helpful reply! Is it not possible to expand this limit or use another way to show all?
